pom.xml

```diff
@@ -22,6 +22,10 @@
 
   <properties>
     <project.build.targetJdk>1.8</project.build.targetJdk>
+    <dep.assertj.version>3.17.1</dep.assertj.version>
+    <dep.plugin.duplicate-finder.version>1.5.1</dep.plugin.duplicate-finder.version>
```

Comment:
Some of these version overrides can go away once we upgrade basepom to 25.8. I tried doing that in this PR, but it automatically enabled prettier, which I didn't want to do as part of this PR (it reformatted all the code)